firebase_admin/_token_gen.py

```diff
@@ -198,7 +200,7 @@ def create_session_cookie(self, id_token, expires_in):
         url = '{0}:createSessionCookie'.format(self.base_url)
         payload = {
             'idToken': id_token,
-            'validDuration': expires_in,
+            'validDuration': str(expires_in),
```

Comment:
I think this is no longer necessary? @yuchenshi should be able to confirm.

Reply:
This should be no longer necessary with the latest Firebase CLI. Please let us know if otherwise. Integration tests can help here.

Reply:
I'll dropped this commit for now, and add it back if the integration tests have a problem (which I don't think will happen).